domains/sly.json

```diff
@@ -0,0 +1,16 @@
+{
+    "description": "Domain name for my personal portfolio website",
+    "repo": "https://github.com/SlyBouhafs/slybouhafs.github.io",
+    "owner": {
+        "username": "SlyBouhafs",
+        "email": "khmedj+github@gmail.com"
+    },
+    "record": {
+        "CNAME": "slybouhafs.github.io",
+        "MX": [
+              "mx1.improvmx.com",
+              "mx2.improvmx.com"
+        ],
+        "TXT": "v=spf1 include:spf.improvmx.com ~all"
```

Comment:
You cannot use CNAME with MX and TXT.

Reply:
Hey, thanks for the feedback!
How do i use it? Do you have to create another file for it?